azext_iot/tests/iothub/configurations/test_iot_config_unit.py

```diff
@@ -899,3 +899,64 @@ def test_config_apply_edge_error(
                 hub_name=mock_target["entity"],
                 content=sample_config_edge_malformed,
             )
+
+
+class TestConfigTestQueries:
+    @pytest.fixture(params=[200])
+    def serviceclient(self, mocker, fixture_ghcs, fixture_sas, request):
+        service_client = mocker.patch(path_service_client)
+        service_client.return_value = build_mock_response(mocker, request.param, {})
+        return service_client
+
+    @pytest.mark.parametrize(
+        "hub_name, target_condition, custom_metric_queries, etag",
+        [
+            (
+                mock_target["entity"],
+                "tags.building=43 and tags.environment='test'",
+                "{\"hi\": \"sd\"}",
+                [generate_generic_id(), None],
+            )
+        ],
+    )
+    def test_config_queries(self, fixture_cmd, serviceclient, hub_name, etag, target_condition, custom_metric_queries):
+        subject.iot_hub_configuration_test_queries(
+            cmd=fixture_cmd,
+            hub_name=hub_name,
```

Comment:
considering that the hub_name will be the same, you can set the variable here rather than parameterizing it